src/wallet/wallet.h

```diff
@@ -145,8 +145,6 @@ static const std::map<std::string,WalletFlags> WALLET_FLAG_MAP{
     {"external_signer", WALLET_FLAG_EXTERNAL_SIGNER}
 };
 
-extern const std::map<uint64_t,std::string> WALLET_FLAG_CAVEATS;
```

Comment:
> so no need to encumber wallet.h and wallet.cpp with it

I'm not sure I totally agree with this, but just because all the flags are defined here already. Like, you'd need to import this header file to use `WALLET_FLAG_AVOID_REUSE` anyway, why not keep the caveats right next to their associated flags?